src/sentry/sentry_metrics/indexer/cache.py

```diff
@@ -3,16 +3,17 @@
 from typing import Mapping, MutableMapping, Optional, Sequence
 
 from django.conf import settings
+from django.core.cache import caches
 
-from sentry.utils.cache import cache
 from sentry.utils.hashlib import md5_text
 
 logger = logging.getLogger(__name__)
 
 
 class StringIndexerCache:
-    def __init__(self, version: int):
+    def __init__(self, version: int, cache_name: str):
         self.version = version
+        self.cache = caches[cache_name]
```

Comment:
@onewland  you probably want to fit this configuration in the indexer instance config schema you built.

Reply:
Here's how the configuration will be used ~ https://github.com/getsentry/getsentry/pull/7885